Milestone 7 : details popup window

zdnahom opened the pull request:
## Milestone 7 : details popup window
In this milestone I implemented : 
- Dynamic project cards.
-  popups for both mobile and desktop screen sizes
- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.

zdnahom committed:
Update index.html file to add popup

zdnahom committed:
Update style.css file to add popup styles

Lornakaboro committed:
Add desktop and mobile popup sections

zdnahom committed:
Update images/ folder to add popup pics

zdnahom committed:
Updat index.js and style.css file to finish detail popup

zdnahom committed:
Update index.js and style.css file for linters pass

zdnahom committed:
Update index.js and style.css to use  anchor buttons

KanzaTahreem commented on index.js:
```diff
@@ -1,25 +1,177 @@
+    </button>
+  </div>
+  <ul class="works-card--info">
+  <li>${openedDetail.info.company_name}</li>
+  <li>
+    <img src="images/works_images/Counter.png" alt="dot icon" />
+  </li>
+  <li>${openedDetail.info.position}</li>
+  <li>
+    <img src="images/works_images/Counter.png" alt="dot icon" />
+  </li>
+  <li>${openedDetail.info.date}</li>
+  </ul>
+</div>
+<img src=${openedDetail.image} alt="project pic" />
+<div class="bottom-part">
+  <p class="works-card--detail">
+    ${openedDetail.description}
+  </p>
+  <div>
+    <ul class="works-card--languages">
+      ${technologies}
+    </ul>
+    <hr />
+    <div class="buttons-container">
+      <a class="portfolio-button" href=${openedDetail.live}>
+        <span>See live</span
+        ><img src="images/live_icon.png" alt="live icon" />
+      </a>
+      <a class="portfolio-button" href=${openedDetail.source}>
+        <span>See source</span
+        ><img src="images/source_icon.png" alt="source icon" />
+      </a>
+    </div>
+  </div>
+</div>
+</div>
+<div class="home_indicator"></div>
+</div>
+</div>`;
```
Great job implementing the project popup window 👍However, I think it would be better to make the project cards and it's content fully responsive while switching from mobile version to desktop version. Right now, the project cards are not behaving according to figma template. For more clarity please see below 🔽

- [ ] All project cards are have different length and style on mobile version. For example, right now, your first popup window's length is different from the rest. Let's make sure that all project popup windows have same style for a better user experience. We don't want our user to have a bad experience? Right 🎯 Please see the video below for more detail 🔽

https://user-images.githubusercontent.com/106119105/216144705-ac200840-69fc-463f-a7b2-cbc55c6487af.mp4

### **_Mobile version 👇_**
- [ ] I think it would be better if you add a little bit more space between the list of technologies and buttons for mobile menu to follow the figma template. What do you think ?🤔
- [ ] Also what do you think of adding the same text for popup as given in figma template? 🤔🥇 If you see the text on project's card is less than the text present on popup window according to figma template. The purpose of the details window is to show more details of our project to the user. Right? Please make sure to add the same text as present in figma template.
- [ ] I think it would be better if you add the space below the project popup window. Right now, there's no space present. Let's make sure to add this one as well. I know you can so that 💪

| Your Design | Figma Design | 
| --- | --- | 
| ![image](https://user-images.githubusercontent.com/106119105/216145859-753eda73-dd60-4b6c-b825-93dd9cbf13d8.png) | ![image](https://user-images.githubusercontent.com/106119105/216142014-4d92d9ad-ccfb-42dd-aef6-7b326e5bac39.png) | 

### **_Desktop version 👇_**
**_Please note that your project popup window looks okay on 50% zoom ratio but for 100% zoom ratio, it is missing some styles. I know you can fix this 💪_**
- [ ] Please make sure that buttons for desktop version are fully responsive while switching from mobile version to desktop version. Right now, from `768px` to `1192px` the buttons are not according to figma template. 
- [ ] Also what do you think of adding the same text for popup as given in figma template? 🤔🥇 If you see the text on project's card is less than the text present on popup window according to figma template. The purpose of the details window is to show more details of our project to the user. Right? Please make sure to add the same text as present in figma template.
- [ ]  I think it would be better if you add a little bit more space between the list of technologies and buttons for mobile menu to follow the figma template. What do you think ?🤔
- [ ] As suggested earlier, I think it would be better if you fix the spacing for the popup window's desktop version according to the figma template as well. Right now you have very less spacing. Also, please make sure to add the space below the project popup window. Right now, there's no space present. Let's make sure to add this one as well. I know you can so that 💪

| Your Design | Figma Design | 
| --- | --- | 
| ![image](https://user-images.githubusercontent.com/106119105/216154668-97d5332e-d55c-46e0-a8b0-c1809bb3e0c3.png) | ![image](https://user-images.githubusercontent.com/106119105/216147334-72ca32d5-efb4-49ef-a3fe-ab1cdb97fb91.png) |

KanzaTahreem reviewed (changes requested):
# Changes Required 🔁

Hi team 👋

Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! 
**_However, I did notice a few areas where some additional changes would further enhance the quality of your project. But you are almost there!_** 💪 🏁

![so_close](https://user-images.githubusercontent.com/99715640/211022680-440a1bb1-4281-4acb-8dbc-3442fe509063.gif)

## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Well done so far following the given Figma design
- ✅ Great job with CSS
- ✅ Descriptive PR and README file
- ✅ Correct Linters

However, there is always room for improvement.

## Required Changes 🛑

> _Please keep in mind that your long hours of work is appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄_

_Please check the comments under the review._

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

zdnahom committed:
Refactor code based on reviewer's suggestion

zdnahom committed:
Update style.css file to give background blur

didierganthier commented on index.js:
```diff
@@ -1,25 +1,175 @@
+</div>
+<img src="images/works_images/detail.png" alt="project pic" />
+<div class="bottom-part">
+  <p class="works-card--detail">
+  Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an  ...
+ </p>
```
[OPTIONAL] Well done on the popup 🎉 but in order to make it look better and cooler 😎  the Modal popup data should be populated with the data from the projects array. 📑 

Although I understand the reason why you did that, probably to make the cards the same size as requested by the previous reviewer. 🤓

zdnahom replied:
@didierganthier Thanks for the review, and I will do that.

didierganthier commented on index.js:
```diff
@@ -1,25 +1,175 @@
+      'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    image: 'images/works_images/desktop_images/desktop1.png',
+    technologies: ['html', 'javascript', 'css', 'python'],
+    live: '#',
+    source: '#',
```
[OPTIONAL] Why not make the project more realistic 🤔 by adding real links for these? Perhaps a link to your live portfolio for the live and a link to the GitHub repo for the source 🔗 . It can be the same link for all of them as these are dummy projects. 😉

didierganthier reviewed (approved):
Hi @zdnahom and @Lornakaboro  ,

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀

<img src="https://media.giphy.com/media/BPJmthQ3YRwD6QqcVD/giphy.gif" />

## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Once again, congratulations on completing this project! 🎉

zdnahom committed:
Update index.js file 

Update index.js file based on reviewer's suggestion . To make project description dynamic and give links to live and source properties.
